VulnerableApplication/Controllers/HomeController.cs

```diff
@@ -30,5 +30,8 @@ public ActionResult Contact()
         public string Hello() {
         return "this is not on the coding rules.";
         }
+        
+        public static string ChangePassword = "PassWord917231";
```

Comment:
[CODE_SMELL] 4 more lines of code need to be covered by tests to reach the minimum threshold of 65.0% lines coverage.